File: java/kattis/JudgingMoose.java
import java.util.Scanner;

public class JudgingMoose {
    /**
https://open.kattis.com/problems/judgingmoose
    */
    public static void main (String[]args){
	Scanner scanInput = new Scanner(System.in);
	int l = scanInput.nextInt();
	int r = scanInput.nextInt();
	if(l == 0 && r == 0){
	    System.out.println("Not a moose");
	} else if( l == r){
	    System.out.println("Even" + " " + (l + r));
	    
	} else {
	    System.out.println("Odd" + " " + (Math.max(l,r) + Math.max(l,r)));
	}
    }
}
	

File: cpp/competitive-programming-4/ch01/1.4 c Selection Only/judgingmoose.cpp
/*
https://open.kattis.com/problems/judgingmoose
Category: 1.4c, Selection Only
Hint: if-else if-else; 3 cases
*/
#include <iostream>
using namespace std;

int main(){
    int l,r;
    cin >> l >> r;
    if(l + r == 0){
        cout << "Not a moose";
    } else if (l == r){
        cout << "Even " << max(l,r) * 2;
    } else {
        cout << "Odd " << max(l,r) * 2;
    }
    return 0;
}
